packages/cms-base/components/listing-filters/SwFilterPrice.vue

```diff
@@ -43,8 +45,12 @@ const prices = reactive<{ min: number; max: number }>({
 });
 
 onMounted(() => {
-  prices.min = Math.floor(props.filter?.min || 0);
-  prices.max = Math.floor(props.filter?.max || 0);
+  prices.min = Math.floor(
+    props.selectedFilters.price.min || props.filter?.min || 0,
```

Comment:
We should use `??` operator because 0 is a valid value

0 ?? 10 -> 0
0 || 10 -> 10

Reply:
agree, changed that